proto/launch/query.proto

```diff
@@ -16,6 +16,11 @@ option go_package = "github.com/tendermint/spn/x/launch/types";
 
 // Query defines the gRPC querier service.
 service Query {
+  // Queries a list of requestSelection items.
+  rpc RequestSelection(QueryRequestSelectionRequest) returns (QueryRequestSelectionResponse) {
+    option (google.api.http).get = "/tendermint/spn/launch/requestSelection";
```

Comment:
```suggestion
    option (google.api.http).get = "/tendermint/spn/launch/requestSelection/{launchID}/{requestIDs}";
```
We should put the rpc def just below `RequestAll` for keeping an intuitive order